styles/scss/image-gallery.scss

```diff
@@ -88,12 +88,9 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
     transition: color .2s ease-out;
   }
 
-  &:hover::before {
+  &:hover::before, &:focus::before {
     color: $ig-blue;
-
-    @media (max-width: $ig-screen-sm-min) {
-      color: $ig-white;
-    }
+    transform: scale(1.1);
```

Comment:
This block of changes adds a focus state that matches the hover state on the fullscreen, play, left, and right navigation buttons so that keyboard users will be able to see the elements as they navigate to them.
I removed the media query, because I couldn't think of any logical scenario where you wouldn't want to still have the focus and hover states, even on a small screen/mobile device, there are users who still use keyboards and pointing devices (though it's a narrow subset of a narrow group). If you can share your initial reasoning, it could be helpful too!
I additionally threw in the small scaling (borrowed from below, where it was applied to only the fullscreen and play buttons). It's not particularly necessary, but additional visual cues are helpful. I'd attach a screenshot, but the only visual difference would be the tiny scale up, the rest just means it shows if you tab between controls.